components/search/DateFilter.vue

```diff
@@ -107,12 +124,24 @@
       },
       end() {
         this.form.end = this.end;
+      },
+      specific() {
+        this.form.specific = this.specific;
       }
     },
 
     methods: {
       emitDateForm() {
-        this.$emit('dateFilter', this.name, this.form);
+        Vue.nextTick(() => { // Change event triggers before v-model has updated. This resolves the issue
```

Comment:
The contents of this function can be reduced to simply:
```js
this.$emit('dateFilter', this.name, this.form);
```
... with the changes to the event handler in the parent component noted below.